src/internal/Selection.js

```diff
@@ -23,7 +23,7 @@ export default class Selection extends React.Component {
       return selectedItems.indexOf(item) === -1;
     });
 
-    if (itemsToAdd.length > 0) {
+    if (selectedItems.indexOf(itemsToAdd) !== -1) {
```

Comment:
In this block, would `itemsToAdd` be an array? Seems like this branch would be unreachable as a result. Do we need it then?

Reply:
yes, `itemsToAdd` is an array that contains `initialSelectedItems` de-inserted, 
so the contents of `itemsToAddd` should not be re-inserted,
I think adding items here must be removed,  but what does do `cWRP` then

Reply:
Potentially nothing. I think the goal here originally was to select things that were added to `initialSelectedItems` but weren't in `selectedItems`. If that branch isn't one that makes sense, can totally remove this 😄